environments/preferences.md

```diff
@@ -62,3 +62,10 @@ Please note that:
 
 You can check whether notifications are enabled and working by click **Test
 Notifications**.
+
+## Autostart
+
+The **Autostart** feature allows you to set the time when your environments will
+automatically build and start. While you can select which environments will
+autostart from the Environments overview page in the Coder UI, you can set the
+specific time here.
```

Comment:
Would it be more clear to replace "Environments overview page" with "create/update Environments modal"?

That way it's not confused with the actual environments page.

Also they are "opting-in" an individual environment vs selecting "which environments".

The latter sounds like I should expect a drop-down of sorts) but it's really just a button they toggle for that one particular environment.